.vscode/extensions.json

```diff
@@ -0,0 +1,4 @@
+{
+    "recommendations": [
```

Comment:
@lascenify when you open the project in VSCode the editor will suggest installing the extensions listed here.